src/js/kits/model-kit-select/search-input.js

```diff
@@ -0,0 +1,36 @@
+import React, { useState } from 'react';
+import { FormattedMessage } from 'react-intl';
+import PropTypes from 'prop-types';
+
+
+const SearchInput = (props) => {
+    const { onChange } = props;
+    const [query, setQuery] = useState('');
+    return (
+        <FormattedMessage
+            id="kits.filter.byName"
+            defaultMessage="filter by name">
+            {
+                placeholder =>
```

Comment:
Nice, didn't know you can wrap your components into `FormattedMessage` to get a message. I used `InjectIntl` wrapper for native elements like input's placeholder. Btw, I think the latest versions of `react-intl` have some hooks for translations.